### PR TITLE
replaced chunks with tuples in validator

### PR DESCRIPTION
**Pull request by KirillLykov**
#### Problem

To address https://github.com/solana-labs/solana/issues/30469

### CODE REVIEW

**Review by CriesofCarrots — COMMENTED** [review]
One nit, otherwise code looks good.
However, we also use the same pattern in `genesis/src/main.rs` (I found them all by grepping for `] => {` 😅 ). Can you please update those as well?
Also, this will probably create conflicts for you https://github.com/solana-labs/solana/pull/30485 . Sorry in advance!

File: validator/src/bin/solana-test-validator.rs [hunk 1]
```diff
@@ -48,6 +48,8 @@ enum Output {
     Dashboard,
 }
 
+use itertools::Itertools;
```
[the remark this answers is not in the document]
Comment (KirillLykov) [reply validator/src/bin/solana-test-validator.rs:51]: interesting

**Comment by KirillLykov** [comment]
> One nit, otherwise code looks good. However, we also use the same pattern in `genesis/src/main.rs` (I found them all by grepping for `] => {` 😅 ). Can you please update those as well? Also, this will probably create conflicts for you #30485 . Sorry in advance!



> One nit, otherwise code looks good. However, we also use the same pattern in `genesis/src/main.rs` (I found them all by grepping for `] => {` 😅 ). Can you please update those as well? Also, this will probably create conflicts for you #30485 . Sorry in advance!

yeah, there are quite some file which use the same pattern, I can put all of them in this PR or only those which look safer that others like genesis, bench-tps and probably some others. Looks pretty safe

**Comment by KirillLykov** [comment]
@CriesofCarrots added genesis

**Review by CriesofCarrots — APPROVED** [review]
Thanks! Lgtm

**Pull request merged**